ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/icons/CalendarMonth.kt

```diff
@@ -0,0 +1,133 @@
+@file:Suppress("MagicNumber", "ObjectPropertyName")
+
+package com.revenuecat.purchases.ui.revenuecatui.icons
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.StrokeJoin
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+
+/**
+ * Exported from [Compose Icons](https://composeicons.com/icons/material-symbols/outlined/calendar_month).
```

Comment:
I decided against including the `androidx.compose.material:material-icons-extended` library, as that one is really large and we're not minifying it properly. Let me know what you think.

Reply:
Hmm I think we would be minifying it? As in, we are only not minifying our SDKs, but not our dependencies I believe... But yeah, if we only need these images, it might be ok to just have them here.

Reply:
Ah you're right! No strong opinion on this. I guess a tiny advantage of not including the library is that we don't increase the size of developers' debug builds.

Reply:
Yeah let's do this for now and we can always revisit later if needed.

Reply:
Yea, agreed!